sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -231,7 +232,7 @@ def from_connection_string(
         auth_timeout: float = 60,
         user_agent: Optional[str] = None,
         retry_total: int = 3,
-        transport_type: Optional["TransportType"] = None,
+        transport_type: TransportType = TransportType.Amqp,
```

Comment:
Default is TransportType.Amqp according to docstring. Passing in None results in error after updating uamqp to compare `TransportType.Amqp.value == transport_type.value`, rather than `TransportType.Amqp == transport_type`.